### ISSUE TITLE
yo this looks pretty cool

### ISSUE THREAD

**Issue by LeehamElectronics** [LABMATT/LMSC#1]
maybe add `.idea `and your `target/` to` .gitignore` ?

_also, first_

**Comment by LABMATT** [comment]
Yeah in a few future projects I now add the system syspsic files to the ignore file but i havent touched LMSC in so long so i havent LOL
